input/docs/getting-started/index.md

```diff
@@ -1,8 +1,8 @@
 Order: 10
 ---
-A big part of understanding ReactiveUI is understanding Reactive Programming, as ReactiveUI is a library built on <a href="https://github.com/dotnet/reactive">Reactive Extensions</a>. Reactive Extensions is a bunch of extension methods for programming in a reactive manner. Reactive programming is programming with asynchronous data streams, called <a href="http://reactivex.io/documentation/observable.html">observables</a>. Unlike tasks, <a href="http://reactivex.io/documentation/observable.html">observables</a> represent one or more values over time. It allows for Linq like operations to mutate the observable event stream and allows for flexible marshalling onto desired threads.
+ReactiveUI is built on a reactive programming foundation and specifically implemented utilizing <a href="https://github.com/dotnet/reactive">Reactive Extensions</a>. <a href="http://reactivex.io/documentation/observable.html">Observables</a>, asynchronous data streams, are core to reactive programming and what makes ReactiveUI impressive. Dealing with Observables is a large part of working with ReactiveUI. Observables can represent one or more values over time. With Observables, a developer can apply LINQ like operations to mutate observable event streams and accomplish their goals more elegantly. Achieve more and worry less, ReactiveUI and the reactive programming model are a perfect choice for most UIs.
 
-When you develop .NET applications, especially large-scale and cross-platform ones, you need to write portable and maintainable code. In XAML-powered platforms, such as Windows Presentation Foundation, Universal Windows Platform, Xamarin Forms and Avalonia UI you can achieve this by implementing the MVVM pattern. MVVM stands for Model-View-ViewModel, where Model represents services, data transfer objects and database entities related to the application domain, View is the UI and ViewModel’s responsibility is to tie these two layers together. ViewModel encapsulates interaction with Model, exposing properties and commands for XAML UI to bind to. 
+A portable and maintainable codebase is important, especially in large-scale and cross-platform .NET implementations. The Model-View-ViewModel pattern, MVVM for short, helps achieve maintainability goals in XAML-powered platforms, such as Windows Presentation Foundation, Universal Windows Platform, Xamarin Forms, and Avalonia UI. With MVVM, the Model represents services, data transfer objects and database entities related to the application domain. The View is the user interface and exposes elements for user interactions. Finally, the ViewModel encapsulates the logic that facilitates communication between the Model and the View.
```

Comment:
can be used in more than just XAML based platforms. Eg Android/iOS xamarin aren't xaml based. I know that was pre-existing before your rewrite but we should probably address it.

Reply:
@glennawatson what do you think would be better there?

> The Model-View-ViewModel pattern, MVVM for short, helps achieve maintainability goals for .NET powered UI frameworks...